### PR TITLE
[16.0][MIG] pos_edit_order_line

### PR DESCRIPTION
**Pull request by flachica**

### CODE REVIEW

File: pos_receipt_hide_price/static/src/xml/OrderReceipt.xml [hunk 1]
```diff
@@ -25,9 +25,9 @@
         <div class="pos-receipt-left-padding">
           x
           <t
-                        t-esc="Math.round(line.quantity * Math.pow(10, env.pos.dp['Product Unit of Measure'])) / Math.pow(10, env.pos.dp['Product Unit of Measure'])"
```
Comment (FrancoMaxime) [inline pos_receipt_hide_price/static/src/xml/OrderReceipt.xml:28]: I think that should be done in another commit because it affects another module

File: pos_edit_order_line/views/pos_config_view.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
         <field name="model">pos.config</field>
         <field name="inherit_id" ref="point_of_sale.pos_config_view_form" />
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='manage_orders']/../.." position="after">
+            <xpath expr="//field[@name='proxy_ip']/../../../.." position="after">
```
Comment (FrancoMaxime) [inline pos_edit_order_line/views/pos_config_view.xml:8]: The configuration system of PoS changed, now its related to res.config.settings. You can see in my merge request how its done now, or in the other modules of the branch 16 that have a config file.

File: pos_edit_order_line/static/src/js/EditOrderButton.js [hunk 3]
```diff
@@ -55,9 +50,6 @@ odoo.define("pos_edit_order_line.EditOrderButton", function (require) {
                     }
                 });
             });
-            if (!_.isEmpty(payload)) {
-                order.trigger("change");
```
Comment (FrancoMaxime) [inline pos_edit_order_line/static/src/js/EditOrderButton.js]: for my information: why is it not needed anymore?

File: pos_edit_order_line/static/src/js/EditOrderPopup.js [hunk 4]
```diff
@@ -1,9 +1,10 @@
...
     "use strict";
 
-    const {useState} = owl.hooks;
+    const {useState} = owl;
     const AbstractAwaitablePopup = require("point_of_sale.AbstractAwaitablePopup");
     const Registries = require("point_of_sale.Registries");
+    const {_t} = require("web.core");
 
     class EditOrderPopup extends AbstractAwaitablePopup {
         constructor() {
```
Comment (FrancoMaxime) [inline pos_edit_order_line/static/src/js/EditOrderPopup.js:10]: Its better to use setup() than constructor() 
Same comment for EditOrderLineInput.js
https://www.odoo.com/documentation/16.0/developer/reference/frontend/owl_components.html

**Review by FrancoMaxime — CHANGES_REQUESTED** [review]

**Comment by flachica** [comment]
@FrancoMaxime  I think in the end it's going to be better to work on your PR. I'm just getting to know OWL and there are some small errors that you have more than solved, do you think I should close this one and let you know so you can change the candidate PR?

**Comment by FrancoMaxime** [comment]
@flachica Ok, I will took some of your improvments and fixe pre-commit inside my pr. Feel free to review https://github.com/OCA/pos/pull/933

**Comment by flachica** [comment]
I am closing this PR in favor of #933
